src/DataTable/composer.json

```diff
@@ -0,0 +1,54 @@
+{
+    "name": "symfony/ux-datatable",
+    "type": "symfony-bundle",
+    "description": "Simple-DataTables integration for Symfony",
+    "keywords": [
+        "symfony-ux"
+    ],
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Matheo Daninos",
+            "email": "matheo.daninos@gmail.com"
+        },
+        {
+            "name": "Thierry Raoux",
+            "email": "thi.raoux@gmail.com"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "autoload": {
+        "psr-4": {
+            "Symfony\\UX\\DataTable\\": "src/"
+        }
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "Symfony\\UX\\DataTable\\Tests\\": "tests/"
+        }
+    },
+    "require": {
+        "php": ">=7.2.5",
```

Comment:
I've browsed some packagist.org repo of symfony/ux* and they are at 95%+ installed on php v8
can't it be the minimum for new package to be merged? also considering php v7 is eol